whois-api/src/main/java/net/ripe/db/whois/api/httpserver/JettyBootstrap.java

```diff
@@ -258,33 +265,51 @@ private Connector createInsecureConnector(final Server server, final HttpConfigu
 
     /**
      * Use the DoSFilter from Jetty for rate limiting: https://www.eclipse.org/jetty/documentation/current/dos-filter.html.
-     * See {@link WhoisDoSFilter} for the customisations added.
-     * @return the rate limiting filter
+     * See {@link DoSLookUpFilter} or {@link DoSUpdateFilter} for the customisations added.
      * @throws JmxException if anything goes wrong JMX wise
      * @throws JMException if anything goes wrong JMX wise
      */
-    private FilterHolder createDosFilter() throws JmxException, JMException {
-        WhoisDoSFilter dosFilter = new WhoisDoSFilter();
-        FilterHolder holder = new FilterHolder(dosFilter);
-        holder.setName("DoSFilter");
-
+    private void createDosFilter(final WebAppContext context) throws JmxException, JMException {
         if (!dosFilterEnabled) {
             LOGGER.info("DoSFilter is *not* enabled");
+            return;
         }
+
+        final String maxRequestPerMsLookUp = "" + 10 * 60 * 1_000; // high default, 10 minutes
+        final FilterHolder lookUpDoSFilter = createDosFilter("DoSLookUpFilter", "50", maxRequestPerMsLookUp, new DoSLookUpFilter());
```

Comment:
What does "50" mean? It's easier to read setters (like below) than passing many arguments in the constructor

Reply:
Should be clear in the lookup tests
just 50 gets are allowed per second. It was like this before.

Reply:
Is it clear now?